README.md

```diff
@@ -30,31 +30,39 @@ docker compose up -d
 Install the [Firebase tools](https://firebase.google.com/docs/emulator-suite):
 
 ```sh
-# Install the Firebase CLI using npm, since the curl script is broken (on 06/09/2024)
-npm install -g firebase-tools
+# Install the Firebase CLI
+curl -sL https://firebase.tools | bash
 
 # Check the version is at least 13.16.0
 firebase --version
 ```
 
 Then copy the `./contrib/firebase-local-data.example` folder to `./firebase-local-data`. This folder will be used to store the local data of the Firebase emulator. It is ignored by git.
+```
+cp -r ./contrib/firebase-local-data.example ./firebase-local-data
+```
 
-Then start it using (replace `GOOGLE_CLOUD_PROJECT` with the value from your `.env` file):
+Then start it using (replace `[GOOGLE_CLOUD_PROJECT]` with the value from your `.env` file):
 
 ```sh
-firebase --project GOOGLE_CLOUD_PROJECT emulators:start --import=./firebase-local-data --export-on-exit
+firebase --project [GOOGLE_CLOUD_PROJECT] emulators:start --import=./firebase-local-data --export-on-exit
 ```
 
 > NB: The `--import` flag is used to import the local data into the emulator. The `--export-on-exit` flag is used to export the data when the emulator is stopped so you don't lose your changes.
 
 ### Launch the project
 
-Export your `.env` file and run the root of the project:
+Export your `.env` file (e.g. using [direnv](https://direnv.net/) or [mise](https://mise.jdx.dev/)) and run the root of the project:
```

Comment:
If we are suggesting mise, maybe add it to the top the file as well, when we instruct to install Go.

Reply:
done

Reply:
done